tools/pd-ut/testdata/group.9.etcdkey

```diff
@@ -89,6 +89,7 @@ resource_group/keyspace/service_limits/ get
 resource_group/keyspace/service_limits/ save
 resource_group/keyspace/settings/ get
 resource_group/keyspace/settings/ watch
+resource_group/keyspace/settings//cross_modify_test save
```

Comment:
Can we remove it? Otherwise, as the tests become more and more, the key will increase.